ipython_widgets/static/widgets/js/widget.js

```diff
@@ -443,7 +441,7 @@ define(["nbextensions/widgets/widgets/js/manager",
     widgetmanager.WidgetManager.register_widget_model('WidgetModel', WidgetModel);
 
 
-    var WidgetView = Backbone.View.extend({
+    var WidgetInterface = {
```

Comment:
Do you mean to be changing the API and inheritance here? Why is that related to removing jquery?